apps/website/src/pages/docs/alert.mdx

```diff
@@ -17,71 +17,51 @@ import * as AllExamples from 'examples';
 
-## Appearance
+## Usage
 
-The alert message must be concise, we recommend no more than 256 characters. If you feel the need to explain in further detail, you may use an optional hyperlink at the end of the alert message to explain more on a new page. You may also include an optional close icon (close icon is required for sticky alerts).
+The alert message must be concise, we recommend no more than 256 characters. If you feel the need to explain in further detail, you may use an optional hyperlink at the end of the alert message to explain more on a new page. You may also include an optional close icon (close icon is required for [sticky](#sticky) alerts).
 
-### Informational
+### Composition API
 
-Default style when there is no defined status for the alert. Inform users about events that they should be aware of, but that are not disruptive to their work.
+You can build Alerts with subcomponents that are fully customizable.
 
-<LiveExample src='Alert.informational.tsx'>
-  <AllExamples.AlertInformationalExample client:load />
-</LiveExample>
-
-### Positive
-
-The positive alert is used to inform the user of a positive status.
-
-<LiveExample src='Alert.positive.tsx'>
-  <AllExamples.AlertPositiveExample client:load />
-</LiveExample>
-
-### Warning
+We provide:
 
-The warning alert is used to inform the user of something that is not currently disruptive to their work, but may be soon or eventually.
+- Alert.Wrapper
+- Alert.Icon
+- Alert.Message
+- Alert.Action
+- Alert.CloseButton
 
-<LiveExample src='Alert.warning.tsx'>
-  <AllExamples.AlertWarningExample client:load />
-</LiveExample>
-
-### Negative
+The type of alert can be specified by utilizing the `type` prop inside `<Alert.Wrapper>`. There are four distinct types of alerts that can be chosen from, namely positive, negative, warning, or informational. Positive alerts are an indication that a task or action has been successfully completed, while warning alerts require attention but do not result in any immediate disruption. Negative alerts, however, signify system outages or failures. Informational alerts are intended to provide users with updates on non-disruptive tasks or work.
 
-Use this only when the system is broken. Leave the technical details behind and explain the problem to the user in an easily understood tone. Mention when the problem might get solved, if it applies.
+By passing the `onClick` prop to `<Alert.Action>` or `<Alert.CloseButton>`, the user can interact with the alert to show additional information or to close. A console message can be logged for event response.
```

Comment:
I feel this can be a separate section (not sure of the location of the section within the page).

We should also mention that Alert.Action is an anchor when `href` is passed and a `button` when `onClick` is passed. Else, users might unnecessarily pass an `as` prop.